mne/viz/tests/test_3d.py

```diff
@@ -389,6 +391,7 @@ def test_limits_to_control_points(renderer):
 
 @testing.requires_testing_data
 @requires_nibabel()
+@requires_pysurfer
```

Comment:
why do you need pysurfer for a test that says use matplotlib?

Reply:
If I remember well, there is no direct call to pysurfer in that function but it ended in the calling stack using a function requiring pysurfer and was crashing the test.